app/lib/notifier.rb

```diff
@@ -24,7 +24,9 @@ def guiders_to_notify
 
   def notify_customer
     if appointment_details_changed?
-      CustomerUpdateJob.perform_later(appointment, CustomerUpdateActivity::UPDATED_MESSAGE)
+      unless appointment.in_the_past?
```

Comment:
I was in two minds on the naming of that, so a second opinion is very welcome. Shall change.